docker/README.rst

```diff
@@ -0,0 +1,173 @@
+Docker Setup for FermiLib + ProjectQ
```

Comment:
FermiLib!? Be careful about that please. You've written FermiLib instead of OpenFermion in several places.

Reply:
I have replaced FermiLib with OpenFermion everywhere